js-String-prototype-concat.md

```diff
@@ -0,0 +1,35 @@
+# String.prototype.concat()
+
+The concat() method combines the text of two or more strings and returns a new string.
+
+**Syntax**
+```js
+str.concat(string2, string3[,..., stringN]);
+```
+
+### Parameters
+
+**string2...string*N***
```

Comment:
Well I was following the MDN docs and they had italicized the N, so I just went with that. It rendered correctly in my editor.